.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull Request Action
+on:
+  push:
+    branches:
+      - feat/bot-*
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: pull-request
+        run: |
+          gh pr edit ${{ github.ref_name }} --title '${{ github.event.head_commit.message }}' --body '${{ github.event.head_commit.message }}' &&
+          gh pr reopen ${{ github.ref_name }} ||
+          gh pr create --base master --title '${{ github.event.head_commit.message }}' --body '${{ github.event.head_commit.message }}'
+  build:
+    uses: ./.github/workflows/build.yml
+    secrets: 
```

Comment:
Is there no way to automatically pass these secrets though?

Reply:
I tried without the secrets which will error out. This is the only way I found that is working at the moment.

Reply:
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows

implies you can "inherit" the secrets between workflows